addon/lib/utils/register-helper.js

```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
+
+function registerHelperIteration1(name, helperFunction) {
+  //earlier versions of ember with htmlbars used this
+  Ember.HTMLBars.helpers[name] = helperFunction;
```

Comment:
yes that would be awesome. (maybe we should consider this some micro-lib we share)